charset_normalizer/md.py

```diff
@@ -153,10 +137,9 @@ def eligible(self, character: str) -> bool:
 
     def feed(self, character: str) -> None:
         if (
-            character not in {"\n", "\t", "\r", "\v"}
+            character.isspace() is False  # includes \n \t \r \v
             and character.isprintable() is False
-            and character.isspace() is False
-            and ord(character) != 0x1A  # Why? Its the ASCII substitute character.
+            and character is not "\x1A"  # Why? Its the ASCII substitute character.
```

Comment:
This is unsafe. There's no guarantee that `character` is identity-wise the same string as `"\x1A"`.

Also, even attempting this syntax will raise a warning:

```
~ $ python3 -c 'foo is not "\x1A"'
<string>:1: SyntaxWarning: "is not" with a literal. Did you mean "!="?
```

Reply:
Agreed :heavy_check_mark: